minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/configuration/ingestors/PullHttpChangeIngestorTest.java

```diff
@@ -55,9 +63,21 @@ public void pullHttpChangeIngestorInit(Properties properties) {
         properties.put(PullHttpChangeIngestor.PORT_KEY, String.valueOf(port));
         properties.put(PullHttpChangeIngestor.HOST_KEY, "localhost");
         properties.put(PullHttpChangeIngestor.PULL_HTTP_POLLING_PERIOD_KEY, "30000");
+        properties.put(PULL_HTTP_BASE_KEY + ".override.core", "true");
+        ConfigurationFileHolder configurationFileHolder = Mockito.mock(ConfigurationFileHolder.class);
+
+        try {
+            ConfigSchema configSchema =
+                SchemaLoader.loadConfigSchemaFromYaml(PullHttpChangeIngestorTest.class.getClassLoader().getResourceAsStream("config.yml"));
+            StringWriter writer = new StringWriter();
+            SchemaLoader.toYaml(configSchema, writer);
+            when(configurationFileHolder.getConfigFileReference()).thenReturn(new AtomicReference<>(ByteBuffer.wrap(writer.toString().getBytes())));
+        } catch (Exception e) {
+            fail("Failed to read test config file", e);
+        }
```

Comment:
As noted, this should be changed to declare an Exception instead of catching it and failing.